consensus/src/chained_bft/block_storage/block_tree.rs

```diff
@@ -50,7 +62,9 @@ pub struct BlockTree<T> {
     /// (including the 3-chain of a voted proposal).
     /// Thus, the structure of `id_to_votes` is as follows:
     /// HashMap<proposed_block_id, HashMap<ledger_info_digest, LedgerInfoWithSignatures>>
-    id_to_votes: HashMap<HashValue, HashMap<HashValue, LedgerInfoWithSignatures>>,
+    id_to_votes: HashMap<HashValue, BlockHash>,
+    // Map of Voting Author to proposed_block_id
```

Comment:
Could you extend the comment to mention that we're managing the latest vote per author and that we're cleaning up the pending votes whenever a new vote is added?

Reply:
nit: May be a better name would be `author_to_last_voted_block_id`. If you find it overly verbose then just `author_to_block_id` :).

Reply:
Done and Done.